specification/gsm/resource-manager/Microsoft.ManagementPartner/preview/2018-02-01/gsm.json

```diff
@@ -0,0 +1,412 @@
+      "properties": {
+        "partnerId": {
+          "type": "string",
+          "description": "This is the partner id"
+        },
+        "tenantId": {
+          "type": "string",
+          "description": "This is the tenant id."
+        },
+        "objectId": {
+          "type": "string",
+          "description": "This is the object id."
+        },
+        "version": {
+          "type": "string",
+          "description": "This is the version."
+        },
+        "updatedTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "This is the DateTime when the partner was updated."
+        },
+        "createdTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "This is the DateTime when the partner was created."
+        },
+        "state": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ManagementPartnerState"
+        }
+      }
+    },
+    "ManagementPartnerState": {
+      "type": "string",
+      "description": "this is the management partner state: Active or Deleted",
+      "enum": [
+        "Active",
+        "Deleted"
+      ]
+    },
+    "ErrorResponseCode": {
+      "type": "string",
+      "description": "this is the error response code that management partner operations may return",
+      "enum": [
+        "NotFound",
+        "Conflict",
+        "BadRequest"
+      ]
+    },
+    "Error": {
+      "type": "object",
+      "description": "this is the management partner operations error",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/ExtendedErrorInfo"
+        }
+      }
+    },
+    "ExtendedErrorInfo": {
+      "type": "object",
+      "description": "this is the extended error info",
+      "properties": {
+        "code": {
+          "$ref": "#/definitions/ErrorResponseCode"
+        },
+        "message": {
+          "type": "string"
+        }
+      }
+    },
+    "OperationList": {
+      "type": "object",
+      "description": "this is the management partner operations list",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/OperationResponse"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Url to get the next page of items."
+        }
+      }
+    },
+    "OperationResponse": {
+      "type": "object",
+      "description": "this is the management partner operations response",
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "display": {
+          "$ref": "#/definitions/OperationDisplay"
+        },
+        "origin": {
+          "type": "string"
+        }
+      }
+    },
+    "OperationDisplay": {
+      "description": "this is the management partner operation",
+      "type": "object",
+      "properties": {
```

Comment:
Description still missing for these properties.

Reply:
@jianghaolu fixed all the issues, can you please help check